### DISCUSSION TITLE
CRAN upload vs Release here

### DISCUSSION THREAD

**Discussion by apeltzer** [metrumresearchgroup/mrgsolve#883] [Ideas]
Hi!

I've noticed that the R package v0.11.2 wasn't uploaded to CRAN - can you please do that? Many users rely on CRAN to be able to install packages more easily and the latest version there is 0.11.1 :-)

Thank you!

x-ref https://github.com/conda-forge/r-mrgsolve-feedstock/pull/2#issuecomment-964922421

**Comment by kylebaron** [comment]
Hi - 

Yes; I'm planning another internal release mid-December and will send that to CRAN once that is completed. Guessing it would get to CRAN some time in January since they likely close over Christmas. 

You can get 0.11.2 [here](https://mpn.metworx.com/docs/snapshots/2021-10-17) in the meantime.

Thanks,
Kyle
